app/src/main/java/tetris/model/GameModel.java

```diff
@@ -63,12 +63,11 @@ public final void setRandomBlock() {
             currentBlock = nextBlock;
         }
 
-        if (!itemCreated && deletedRowCount != 0 && deletedRowCount % 10 == 0) {
-            itemCreated = true;
+        if (itemCount >= 2) {
+            itemCount = 0;
```

Comment:
```suggestion
        if (itemCount >= 10) {
            itemCount -= 10;
```
10개 줄을 삭제할 때마다 이므로 이렇게 바꿔야 할 것 같고, 9줄에서 3줄이 한번에 지워져서 12줄이 되었을 경우 카운트가 0부터 다시 시작이 아닌 12 - 10, 2부터 시작하게끔 해야 할 것 같네요.